src/resources/js/admin-upgrade-page.js

```diff
@@ -0,0 +1,39 @@
+tribe.upgradePage =  tribe.upgradePage || {};
+
+( function ( $, obj ) {
+	'use strict';
+
+	obj.setup = function() {
+		if ( $( '#current-settings-tab' ).val() != 'upgrade' ) {
+			// if it is already enabled, we don't need to show the button
+			if ( tribe_upgrade.v2_is_enabled == "1" ) {
+				return;
+			}
+
+			$( '.tribe_settings > h1' ).append( '<button id="upgrade-button">✨ Upgrade your calendar views</button>' );
```

Comment:
These should come from the PHP side since the string here needs to be translated, we should include it as a hidden element and only move it with JS.

Reply:
I changed it to a localize so it can come through. There is no _easy_ way to add this on the PHP side, so, this seems like the best path for now.